tests/rdp.rs

```diff
@@ -77,10 +77,13 @@ fn run_client() -> Child {
         client_command.args(&["--domain", domain]);
     }
 
-    client_command.spawn().expect("failed to run IronRDP client")
+    client_command.spawn().unwrap_or_else(|e| panic!("{:?}", e))
```

Comment:
Does this correspond to some clippy warning? `expect` (like it was previously) or `unwrap` (equivalent to the current code but less contrived) should be good enough.
```suggestion
    client_command.spawn().unwrap()
```

Reply:
Thank you, this change is redundant indeed, I am accidentally committed it. I will return `expect` back.